src/systems/spacecraft_thruster_model/SpacecraftThrusterModel.hh

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2019 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef GZ_SIM_SYSTEMS_SPACECRAFTTHRUSTERMODEL_HH_
+#define GZ_SIM_SYSTEMS_SPACECRAFTTHRUSTERMODEL_HH_
+
+#include <gz/sim/System.hh>
+#include <memory>
+
+namespace gz
+{
+namespace sim
+{
+// Inline bracket to help doxygen filtering.
+inline namespace GZ_SIM_VERSION_NAMESPACE {
+namespace systems
+{
+  // Forward declaration
+  class SpacecraftThrusterModelPrivate;
+
+  /// \brief This system applies a thrust force to models with RCS-like
```

Comment:
I may have mentioned it before but document the SDF parameters here and topics that control this plugin. Also point to some citation explaining where this model comes from.